modulemd/include/private/modulemd-util.h

```diff
@@ -63,49 +129,195 @@ G_DEFINE_AUTOPTR_CLEANUP_FUNC (modulemd_tracer, modulemd_trace_free);
 
 G_END_DECLS
 
-
+/**
+ * modulemd_hash_table_deep_str_copy:
+ * @orig: A #GHashTable to copy, containing string keys and values.
```

Comment:
```suggestion
 * @orig: A #GHashTable to copy, containing string keys and string values.
```
I think it's better to be overly-explicit.